collections/_development_standards/patterns.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title:  "Patterns"
+category: development_standards
+---
+
+## Development Patterns
+
+Below are described the major engineering patterns currently in-use across the Service. There is no definitive list as it is understood that patterns should be used when and where appropriate.
+
+### Orchestrator
+
+As controllers age they are asked to do more and more work which can result in serious bloat. We want to keep our controllers focused on what they should be doing such as handling requests, invoking business rules and returning responses. So to keep them light we need to hand-off any other work that may end up in a controller out to somewhere else and the Orchestrator can be used to fill this need. 
+
+The use of Orchestrators can also make a codebase more testable by moving logic and flow out of controllers and their tight coupling to the runtime.
+
+![Orchestrator Overview](./images/OrchestratorOverview.png)
```

Comment:
We have moved away from this pattern over time of creating separate orchestrator classes - the controller actions are normally very light weight anyway and only tend to have calls to mediatr and then building a view model from it. All tests are isolated that way and the tests on the controller are normally very light weight, letting the controller do its intended orchestrator role.

Reply:
That makes sense to me. I'll leave in the Orchestrator section for historical purposes and add comments around the reasoning behind move away from the pattern.

I do have some reservations about Mediatr though as I think it makes the code harder to read